L1Trigger/L1THGCal/test/HGCalTriggerGeomTesterV9Imp2.cc

```diff
@@ -854,6 +1009,14 @@ void HGCalTriggerGeomTesterV9Imp2::fillTriggerGeometry()
       moduleIEta_ = sc_id.ietaAbs();
       moduleIPhi_ = sc_id.iphi();
       module_ = 0;
+    } else if (id.det() == DetId::Forward and id.subdetId() == ForwardSubdetector::HFNose) {
+      HGCalDetId nose_id(module_triggercells.first);
```

Comment:
This is not correct for Nose modules, right?

Reply:
fixed, HFNoseTriggerDetId nose_id(module_triggercells.first);